packages/core/test/integration/query-interface/schemas.test.ts

```diff
@@ -1,14 +1,23 @@
 import { expect } from 'chai';
 import { spy } from 'sinon';
-import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
+import type { CreateSchemaQueryOptions } from '../../../src/dialects/abstract/query-generator';
```

Comment:
```suggestion
import type { CreateSchemaQueryOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/dialects/abstract/query-generator';
```

This should remain working afaik

Reply:
Those worked locally - thank you!